code/game/atoms_movable.dm

```diff
@@ -45,13 +45,7 @@
 		loc.handle_atom_del(src)
 	for(var/atom/movable/AM in contents)
 		qdel(AM)
-	var/turf/un_opaque
-	if(opacity && isturf(loc))
-		un_opaque = loc
-
 	loc = null
-	if(un_opaque)
-		un_opaque.recalc_atom_opacity()

```

Comment:
Just something I noticed; this is handled in the lighting system now; we don't need this anymore.